armi/bookkeeping/db/databaseInterface.py

```diff
@@ -151,36 +151,24 @@ def interactEveryNode(self, cycle, node):
         if self.o.cs["tightCoupling"]:
             # h5 cant handle overwriting so we skip here and write once the tight coupling loop has completed
             return
-        self.writeDBEveryNode(cycle, node)
+        self.writeDBEveryNode()
 
-    def writeDBEveryNode(self, cycle, node):
```

Comment:
sending unused args in is unclear and frustrating for debugging. got rid of unsued.

Reply:
I can confirm that no one calls this method outside ARMI.  I checked Nala and all submodules, Pylot, and all our benchmark reactors.

That makes this API change safely contained inside ARMI.